PushNotifications.podspec

```diff
@@ -7,9 +7,11 @@ Pod::Spec.new do |s|
   s.author           = { "Luka Bratos" => "luka@pusher.com" }
   s.source           = { git: "https://github.com/pusher/push-notifications-swift.git", tag: s.version.to_s }
   s.social_media_url = 'https://twitter.com/pusher'
+  s.documentation_url = ''
```

Comment:
Is this needed?

Reply:
This sneaked in unintentionally. I wanted to do a separate PR for this.
But if it's in already now I might as well put a link to the docs in 👍